### PR TITLE
feat: make command line more consistent and log bypass options in messages

### PR DESCRIPTION
**Pull request by wandyezj**
add `--no-verify` to be consistent with other no options.

log bypass option for:
--no-verify
--allow-star-activation
--allow-missing-repository

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -183,6 +183,7 @@ module.exports = function (argv: string[]): void {
 		.option('--yarn', 'Use yarn instead of npm (default inferred from presence of yarn.lock or .yarnrc)')
 		.option('--no-yarn', 'Use npm instead of yarn (default inferred from lack of yarn.lock or .yarnrc)')
 		.option('--noVerify')
+		.option('--no-verify')
```
Comment (wandyezj) [inline src/main.ts:186]: `--no-verify` is linked to using proposed APIs. Would it make sense to change this option to `--allow-proposed-apis` to match the other allow options?

File: src/main.ts [hunk 2]
```diff
@@ -183,6 +183,7 @@ module.exports = function (argv: string[]): void {
 		.option('--yarn', 'Use yarn instead of npm (default inferred from presence of yarn.lock or .yarnrc)')
 		.option('--no-yarn', 'Use npm instead of yarn (default inferred from lack of yarn.lock or .yarnrc)')
 		.option('--noVerify')
```
Comment (wandyezj) [inline src/main.ts:185]: There does not seem to be a convenient way to hide an existing option from help.

Comment (wandyezj) [inline src/main.ts:185]: Can hide the old option with: https://github.com/tj/commander.js/issues/1716#issuecomment-1099615098

Comment (joaomoreno) [reply src/main.ts:185]: Yup I suggest we do this.

File: src/main.ts (hunk 1 above)
Comment (joaomoreno) [reply src/main.ts:186]: Yeah I think that makes sense as well.

**Review by joaomoreno — CHANGES_REQUESTED** [review]

**Review by joaomoreno — APPROVED** [review]

**Comment by joaomoreno** [comment]
Thanks! 🍻

**Pull request merged**